config/config.go

```diff
@@ -376,6 +376,7 @@ type DeploymentConfig struct {
 type GraphFindOption struct {
 	Description string `yaml:"description,omitempty" json:"description,omitempty"`
 	Expression  string `yaml:"expression,omitempty" json:"expression,omitempty"`
+	AutoEnable  bool   `yaml:"auto_enable,omitempty" json:"autoEnable,omitempty"`
```

Comment:
See the operator PR review, propose to change this to "AutoSelect".

Reply:
Thanks for the review, @jshaughn ! I have updated the changes.